core.multiview/src/org/netbeans/core/multiview/SplitLayerUI.java

```diff
@@ -93,7 +93,7 @@ public void eventDispatched( AWTEvent event ) {
                         @Override
                         public void run() {
                             TopComponent tc = (TopComponent)SwingUtilities.getAncestorOfClass( TopComponent.class, content );
-                            SplitAction.splitWindow( tc, orientation, splitLocation );
+                            AbstractSplitDocumentAction.splitWindow(tc, orientation, splitLocation);
```

Comment:
... why not leave the original call, it still works

Reply:
Fixed